sa/sa_test.go

```diff
@@ -2209,3 +2209,56 @@ func TestGetAuthorizationsFast(t *testing.T) {
 		t.Errorf("Wrong challenge fetch count: expected 2, got %d", challengeFetchCount)
 	}
 }
+
+func TestUpdateChallengesPendingOnly(t *testing.T) {
+	sa, fc, cleanUp := initSA(t)
+	defer cleanUp()
+
+	expires := fc.Now().Add(time.Hour)
+	ctx := context.Background()
+
+	// Create a registration to work with
+	reg := satest.CreateWorkingRegistration(t, sa)
+
+	// Create a pending authz
+	input := core.Authorization{
+		RegistrationID: reg.ID,
+		Expires:        &expires,
+		Status:         core.StatusPending,
+		Identifier:     core.AcmeIdentifier{Type: core.IdentifierDNS, Value: "example.com"},
+		Challenges: []core.Challenge{
+			core.Challenge{
+				Type:   "http-01",
+				Status: "pending",
+			},
+		},
+	}
+	authz, err := sa.NewPendingAuthorization(ctx, input)
+	test.AssertNotError(t, err, "Couldn't create new pending authorization")
+
+	authz.Status = core.StatusValid
+	authz.Challenges[0].Status = core.StatusValid
+	err = sa.FinalizeAuthorization(ctx, authz)
+	test.AssertNotError(t, err, "Couldn't finalize pending authorization with ID "+authz.ID)
+
+	r, err := sa.GetAuthorization(ctx, authz.ID)
```

Comment:
I think this needs to be changed to `_, err = sa.GetAuthorization(ctx, authz.ID)` to fix the following error in CI:
```
sa/sa_test.go:2244:2: r declared and not used
```